ConferencesApp/Base.lproj/Main.storyboard

```diff
@@ -15,9 +15,15 @@
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <animations/>
                         <color key="backgroundColor" white="1" alpha="1" colorSpace="calibratedWhite"/>
+                        <searchBar key="tableHeaderView" contentMode="redraw" placeholder="Name or Location" id="5UF-uC-8w4">
```

Comment:
May be we can just have `Search` as a placeholder so we can add more data into search later.. like `tags` and twitter handle.